product_attribute_value_archive/tests/test_archive_attribute_value.py

```diff
@@ -75,18 +74,22 @@ def setUpClassCommon(cls):
             ]
         )
         cls.red_pen = pen_template_value.ptav_product_variant_ids
+        cls.order = cls.env.ref("sale.sale_order_1")
```

Comment:
Then this line deserves a comment:
```python
# NOTE: use an existing order and avoid the creation a new one
# to avoid NOT NULL constraint introduced by other addons
```